resources/functionMap.php

```diff
@@ -12639,7 +12639,7 @@
 'unixtojd' => ['int|false', 'timestamp='=>'int'],
 'unlink' => ['bool', 'filename'=>'string', 'context='=>'resource'],
 'unpack' => ['array|false', 'format'=>'string', 'data'=>'string', 'offset='=>'int'],
-'unregister_tick_function' => ['void', 'function_name'=>'string'],
+'unregister_tick_function' => ['void', 'callback'=>'callable'],
```

Comment:
The parameter does not need to be renamed

Reply:
Okay, sounds good. I just looked in the old manual . In PHP 7, it was called "$function_name" , and in PHP 8 it was renamed to "$callback" (so it should be renamed in functionMap_php80delta.php then)

Reply:
This doesn't need to be done, the PHP 8 signatures are mostly taken from https://github.com/phpstan/php-8-stubs/blob/main/stubs/ext/standard/unregister_tick_function.php already.